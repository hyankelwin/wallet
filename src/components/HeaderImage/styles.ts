import styled from 'styled-components';
import img from '../../assets/logos/logo.png';

export const Content = styled.div`
  height: 100px;
  width: 200px;
  background-size: contain;
  margin-top: 20px;
  background-image: url(${img});
  background-size: contain;
  background-repeat: no-repeat;
`;
