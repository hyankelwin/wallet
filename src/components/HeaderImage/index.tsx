import React from 'react';
import { Content } from './styles';

const HeaderImage: React.FC = () => {
  return <Content />;
};

export default HeaderImage;
