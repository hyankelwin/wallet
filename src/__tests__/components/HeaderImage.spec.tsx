import { mount } from 'enzyme';
import HeaderImage from '../../components/HeaderImage';

describe('HeaderImage component', () => {
  it('should be able to render HeaderImage component with success', () => {
    const wrapper = mount(<HeaderImage />);

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
