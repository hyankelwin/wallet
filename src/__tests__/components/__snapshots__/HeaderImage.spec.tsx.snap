// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderImage component should be able to render HeaderImage component with success 1`] = `
"<HeaderImage>
  <styled.div>
    <div className=\\"sc-bdvvtL eSCykS\\" />
  </styled.div>
</HeaderImage>"
`;
